components/ReplyItem.js

```diff
@@ -1,55 +1,426 @@
+        <Avatar user={user} size={72} />
+        {/*
+          <div title={TYPE_NAME[replyType]}>{TYPE_ICON[replyType]}</div>
+        */}
+      </Box>
+      <Box pr="14px" display={['block', 'block', 'none']}>
+        <Avatar user={user} size={30} />
+        {/*
+          <div title={TYPE_NAME[replyType]}>{TYPE_ICON[replyType]}</div>
+        */}
+      </Box>
+      <Box py="12px" flexGrow={1}>
+        <div
+          className={classes.replyType}
+        >{t`${user.name} consider this ${TYPE_NAME[replyType]}`}</div>
+        <TextExpansion content={text} />
+        <div className={classes.status}>
+          <div className={classes.actions}>
+            <button
+              className={classes.vote}
+              type="button"
+              onClick={e => openVotePopover(e, 'UPVOTE')}
+            >
+              <img className={classes.thumbUp} src={ThumbUp} alt="thumb-up" />
+            </button>
+            <button
+              className={classes.vote}
+              type="button"
+              onClick={e => openVotePopover(e, 'DOWNVOTE')}
+            >
+              <img
+                className={classes.thumbDown}
+                src={ThumbDown}
+                alt="thumb-down"
+              />
+            </button>
+            <div className={classes.buttonGroup}>
+              <button className={classes.vote} type="button">
+                {positiveFeedbackCount}
+                <img
+                  className={classes.thumbUp}
+                  src={ThumbUpSolid}
+                  alt="thumb-up"
+                />
+              </button>
+              <button className={classes.vote} type="button">
+                {negativeFeedbackCount}
+                <img
+                  className={classes.thumbDown}
+                  src={ThumbDownSolid}
+                  alt="thumb-down"
+                />
+              </button>
+              <button
+                className={classes.vote}
+                type="button"
+                onClick={openReasonsPopover}
+              >
+                {t`See Reasons`}
+              </button>
+            </div>
           </div>
+          {isValid(createdAt) && (
+            <span
+              className={classes.createdAt}
+              title={format(createdAt)}
+            >{t`${timeAgoStr} ago`}</span>
+          )}
+        </div>
+      </Box>
+      <Popover
+        id={`reply-reasons-${reply.id}`}
+        open={!!reasonsPopoverAnchorEl}
+        anchorEl={reasonsPopoverAnchorEl}
+        onClose={closeReasonsPopover}
+        anchorOrigin={{
+          vertical: 'top',
+          horizontal: 'center',
+        }}
+        classes={{ paper: classes.popover }}
+      >
+        <button
+          type="button"
+          className={classes.closeButton}
+          onClick={closeReasonsPopover}
+        >
+          <CloseIcon />
+        </button>
+        {text}
+        <div className={classes.tabs}>
```

Comment:
because that for the current mockup, there will be custom svg icons in tab, and I found it might be difficult adapting tabs of material-ui in this case

Reply:
Hmm but Material-UI tabs already support icons + text, it should be doable by customizing Tab's `wrapper` to make it align horizontally, or use `position: absolute` on icons and use right-padding on `wrapper` to make space.

![image](https://user-images.githubusercontent.com/108608/80286205-a00a6c00-875c-11ea-8761-b931fdf5c1e4.png)